datafusion/core/tests/sqllogictests/test_files/window.slt

```diff
@@ -2036,6 +2036,184 @@ SELECT
 -1114 -1927628110
 15673 -1899175111
 
+# test_window_agg_partition_by_set
+statement ok
+set datafusion.execution.target_partitions = 1;
+
+query TT
+EXPLAIN SELECT
```

Comment:
This test checks for whether `BoundedWindowAggExec` treats partition by expressions as set. Physical plan shouldn't have `SortExec` in between `BoundedWindowAggExec`